crates/meroctl/src/cli/context/identity/alias.rs

```diff
@@ -87,6 +89,17 @@ pub enum ContextIdentityAliasSubcommand {
         #[arg(long, short, default_value = "default")]
         context: Alias<ContextId>,
     },
+
+    #[command(about = "List all the aliases under the context")]
+    List {
+        #[arg(help = "The context whose aliases need to be listed")]
+        #[arg(long, short, default_value = "default")]
+        context: Alias<ContextId>,
+
+        #[arg(help = "If all the aliases for all the contexts need to listed")]
+        #[arg(long, default_value = "false", conflicts_with = "context")]
+        all: bool,
```

Comment:
`--all` was supposed to be meant for `context identity list`, we can remove it for `context identity alias list`, should they need all aliases across all contexts, a little shell scripting won't hurt

Reply:
after you do this, could you update `merod`'s `context identity alias list` to not be `Option`? it should use `default_value` instead, thanks!